src/value.rs

```diff
@@ -432,12 +432,11 @@ pub mod time_format {
 
     pub static FORMAT_DATE: &[FormatItem<'static>] = format_description!("[year]-[month]-[day]");
     pub static FORMAT_TIME: &[FormatItem<'static>] =
-        format_description!("[hour]:[minute]:[second]");
+        format_description!("[hour]:[minute]:[second].[subsecond]");
     pub static FORMAT_DATETIME: &[FormatItem<'static>] =
-        format_description!("[year]-[month]-[day] [hour]:[minute]:[second]");
-    pub static FORMAT_DATETIME_TZ: &[FormatItem<'static>] = format_description!(
-        "[year]-[month]-[day] [hour]:[minute]:[second] [offset_hour sign:mandatory][offset_minute]"
-    );
+        format_description!("[year]-[month]-[day] [hour]:[minute]:[second].[subsecond]");
+    pub static FORMAT_DATETIME_TZ: &[FormatItem<'static>] =
+        format_description!("[year]-[month]-[day] [hour]:[minute]:[second].[subsecond]Z");
```

Comment:
Hey @kyoto7250, I don't think we need to change this part of the code. This is just for `to_string` on SQL statement and it has nothing to do with value binding. I think we should keep it as-is.

Reply:
Thoughts? @kyoto7250